components/collector/src/source_collectors/bitbucket/merge_requests.py

```diff
@@ -0,0 +1,68 @@
+"""Bitbucket merge requests collector."""
+
+from typing import cast
+
+from collector_utilities.functions import match_string_or_regular_expression
+from collector_utilities.type import URL
+from model import Entities, Entity, SourceResponses
+
+from .base import BitbucketProjectBase
+
+
+class BitbucketMergeRequests(BitbucketProjectBase):
+    """Collector for merge requests in Bitbucket."""
+
+    PAGE_SIZE = 100  # Page size for Bitbucket pagination
```

Comment:
`BitbucketInactiveBranches` also has this class variable. Please pull it up to the common base class `BitbucketBase`.